src/pages/Dashboard/Dashboard.jsx

```diff
@@ -61,24 +72,35 @@ export default function Dashboard() {
             </TableHead>
 
             <TableBody>
-              {tasks.tasks.map((task, index) => (
-                <TableRow key={index}>
-                  <TableCell align='center'>{task.title}</TableCell>
-                  <TableCell align='center'>{task.description}</TableCell>
-                  <TableCell align='center'>{task.status}</TableCell>
-                  <TableCell align='center'>
-                    <IconButton
-                      size='large'
-                      onClick={() => handleDeleteTask(task.id)}
-                    >
-                      <DeleteIcon />
-                    </IconButton>
-                  </TableCell>
-                </TableRow>
-              ))}
+              {tasks.tasks
+                .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
+                .map((task, index) => (
+                  <TableRow key={index}>
+                    <TableCell align='center'>{task.title}</TableCell>
+                    <TableCell align='center'>{task.description}</TableCell>
+                    <TableCell align='center'>{task.status}</TableCell>
+                    <TableCell align='center'>
+                      <IconButton
+                        size='large'
+                        onClick={() => handleDeleteTask(task.id)}
+                      >
+                        <DeleteIcon />
+                      </IconButton>
+                    </TableCell>
+                  </TableRow>
+                ))}
             </TableBody>
           </Table>
         </TableContainer>
+        <TablePagination
+          component='div'
+          count={tasks.tasks.length}
+          onPageChange={handleChangePage}
+          page={page}
+          rowsPerPageOptions={[10, 25, 50]}
+          rowsPerPage={rowsPerPage}
+          onRowsPerPageChange={handleChangeRowsPerPage}
+        />
         <AddTask></AddTask>
```

Comment:
[Recommendation] If we do not have a child component, it is preferable to use self-closing tag on the component.
- https://reactjs.org/docs/jsx-in-depth.html#children-in-jsx
- https://react-cn.github.io/react/tips/self-closing-tag.html#:~:text=All%20tags%20must%20be%20closed,div%3E%20is%20also%20an%20equivalent.
- https://stackoverflow.com/questions/48991212/react-component-closing-tag